### PR TITLE
chore: allows daily dependabot version updates for @faststore/* deps

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

This PR is the first step of the FastStore Evergreen Dependabot.
Screen Shot 2022-12-20 at 18 49 16

This PR aims to enable daily [dependabot version updates](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates) for the `@faststore/*` dependencies.

We can deal with dependabot at first, but after we can try other multi-platform tools for automated dependency updates, like [renovate](https://docs.renovatebot.com/#supported-platforms).

## How does it work?

Based on the Github Docs previously mentioned. We can enable the version updates by creating the `dependabot.yml` file in the `.github` folder with [custom configs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file).

GitHub also allows us to enable `dependabot` version updates directly from the settings, but this is enable once we upload the `dependabot.yml` file, as mentioned previously.
Screen Shot 2022-12-13 at 20 50 59

## How to test it?

For testing purposes, We created a new lib called [@test-fs/test-gh-dependabot-updates](http://www.npmjs.com/package/@test-fs/test-gh-dependabot-updates), and used it as a dependency in the [vtexgame1.store](https://github.com/vtex-sites/vtexgame1.store/blob/main/.github/dependabot.yml) repo. 
After that, we add a similar `dependabot` config file there and observed that `dependabot` opens the PRs as expected.
PRs opened by dependabot:
https://github.com/vtex-sites/vtexgame1.store/pull/16
https://github.com/vtex-sites/vtexgame1.store/pull/15
https://github.com/vtex-sites/vtexgame1.store/pull/14
https://github.com/vtex-sites/vtexgame1.store/pull/13

Why do we use other repos and libs?
<strike>- Webops doesn't support builds from the Evergreen approach using `.faststore` temporary folder.</strike> now it works after https://github.com/vtex-sites/starter.store/pull/5

## References
https://github.blog/changelog/2022-06-02-enable-dependabot-version-updates-from-the-repository-settings-page/
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
https://www.npmjs.com/package/@faststore/core
https://docs.renovatebot.com/#supported-platforms

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    pull-request-branch-name:
+      # Separate sections of the branch name with a hyphen
+      # for example, `dependabot-npm_and_yarn-faststore-core-1.0.0`
+      separator: "-"
```
Comment (eduardoformiga) [inline .github/dependabot.yml:10]: this is because webops didn't work with multiples '/' in the branch name.

**Review by ArthurTriis1 — APPROVED** [review]
LGTM

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (icazevedo) [inline .github/dependabot.yml:6]: What if we have to release a hot fix? Is there a way to force a scan?

**Review by icazevedo — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (eduardoformiga) [reply .github/dependabot.yml:11]: done [here](https://github.com/vtex-sites/starter.store/pull/5/commits/2d2a2947086247558cfb3a04612dd38fdc989394).

File: .github/dependabot.yml (hunk 2 above)
Comment (eduardoformiga) [reply .github/dependabot.yml:6]: As we are just updating the minor and patch versions without the `auto-merge` enabled by default, the dev team from the agency is supposed to test before merge.

Using dependabot we just have [daily, weekly, and monthly](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval) updates. 

The way to force a scan is manually using this `check for updates` option
Screen Shot 2023-02-13 at 19 44 30

Something to pay attention to is this issue https://github.com/dependabot/dependabot-core/issues/3488 (maybe in the future).

**Pull request merged**